### PR TITLE
Remove update operators from market contract

### PR DESCRIPTION
**Pull request by emmanueldenloye**
This should resolve issue #130

### CODE REVIEW

File: contracts/ligo/src/fixed_price_sale_market.mligo [hunk 1]
```diff
@@ -89,16 +89,14 @@ let buy_token(sale, storage: sale_param * storage) : (operation list * storage)
   in
   let tx_money_op = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale.price, Tezos.sender, seller) in
   let tx_nft_op = transfer_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
-  let tx_nft_remove_operator_op = remove_operator_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
```
Comment (emishur) [inline contracts/ligo/src/fixed_price_sale_market.mligo:92]: consider removing dead code: `remove_operator_nft` and `add_operator_nft` functions

File: contracts/ligo/src/fixed_price_sale_market.mligo [hunk 2]
```diff
@@ -89,16 +89,14 @@ let buy_token(sale, storage: sale_param * storage) : (operation list * storage)
...
   let new_s = Big_map.remove sale storage in
-  (tx_money_op :: tx_nft_remove_operator_op :: tx_nft_op :: []), new_s
```
Comment (emishur) [inline contracts/ligo/src/fixed_price_sale_market.mligo:94]: also add/remove operator should happened before/after market contract is invoked.
I did not verify test or actual code that interacts with the market contract

Comment (emmanueldenloye) [reply contracts/ligo/src/fixed_price_sale_market.mligo:94]: I'll add the test to this branch. I've only added tests for the market contract that sells NFTs in tez.

**Comment by emishur** [comment]
Same changes need to be done for `fixed_price_sale_market_tez.mligo`

**Review by emishur — APPROVED** [review]

**Pull request merged**